libs/dh/admin/feature-user-management/src/lib/overview/dh-users-overview.component.html

```diff
@@ -63,6 +63,4 @@ <h3>{{ t("tabLabel") }}</h3>
   </vater-flex>
 </watt-card>
 
-@if (isInviteUserModalVisible) {
-  <dh-invite-user-modal (closed)="modalOnClose()" />
-}
+<dh-invite-user-modal />
```

Comment:
issue: a consequence of this change is that the `dh-invite-user-modal` component will be initialized as soon as the user navigates to the `/admin/users` page, not when the "Invite" button was clicked (as it was before). This is probably fine. Let's be aware of it.